pkg/validation/internal/community.go

```diff
@@ -0,0 +1,277 @@
+				if !strings.Contains(line[i], "=") {
+					checks.errs = append(checks.errs, fmt.Errorf("this bundle is using the deprecated "+
+						"apis for CRDs (apiextensions.k8s.io/v1beta1) and an invalid syntax (%s) on the LABEL %s."+
+						"Please, upgrade your project to no longer use the deprecated APIs or change this configuration. "+
+						"(e.g. LABEL %s='4.6-4.8')",
+						line[i], ocpLabelindex, ocpLabelindex))
+					return checks
+				}
+
+				value := strings.Split(line[i], "=")
+				indexRange := value[1]
+				doubleCote := "\""
+				singleCote := "'"
+				indexRange = strings.ReplaceAll(indexRange, singleCote, "")
+				indexRange = strings.ReplaceAll(indexRange, doubleCote, "")
+				if len(indexRange) > 1 {
+					// if has the = then, the value needs to be < 4.9
+					if strings.Contains(indexRange, "=") {
+						version := strings.Split(indexRange, "=")[1]
+						verParsed, err := semver.ParseTolerant(version)
+						if err != nil {
+							checks.errs = append(checks.errs, fmt.Errorf("unable to parse the value (%s) on (%s)",
+								version, ocpLabelindex))
+							return checks
+						}
+
+						if verParsed.GE(semVerOCPV1beta1Unsupported) {
+							checks.errs = append(checks.errs, fmt.Errorf("this bundle is using the deprecated "+
+								"apis for CRDs (apiextensions.k8s.io/v1beta1) and is configured via the LABEL (%s) "+
+								"to be distributed on (%s) where this kubernetes apis is no longer supported. Please, "+
+								"upgrade your project to no longer use the deprecated APIs or change this configuration. "+
+								"(e.g. LABEL %s='4.6-4.8')",
+								ocpLabelindex, version, ocpLabelindex))
+							return checks
+						}
+					} else {
+						// if not has not the = then the value needs contains - value less < 4.9
+						if !strings.Contains(indexRange, "-") {
+							checks.errs = append(checks.errs, fmt.Errorf("this bundle is using the deprecated "+
+								"APIs for CRDs (apiextensions.k8s.io/v1beta1) and is configured via the LABEL (%s) "+
+								"with (%s) when it cannot be distributed to version(s) >= %s. Please, upgrade your "+
+								"project to no longer use the deprecated APIs or change this configuration. "+
+								"(e.g. LABEL %s='4.6-4.8')",
+								ocpLabelindex, indexRange, ocpVerV1beta1Unsupported, ocpLabelindex))
+							return checks
+						}
+
+						version := strings.Split(indexRange, "-")[1]
+						verParsed, err := semver.ParseTolerant(version)
+						if err != nil {
+							checks.errs = append(checks.errs, fmt.Errorf("unable to parse the value (%s) on (%s)",
+								version, ocpLabelindex))
+							return checks
+						}
+
+						if verParsed.GE(semVerOCPV1beta1Unsupported) {
+							checks.errs = append(checks.errs, fmt.Errorf("this bundle is using the deprecated "+
+								"APIs for CRDs (apiextensions.k8s.io/v1beta1) and is configured via the LABEL (%s) "+
+								"to be distributed on (%s) where this kubernetes APIs is no longer supported. Please, "+
```

Comment:
Tense doesn't match. Either "these Kubernetes APIs are no longer" or "this Kubernetes API is no longer".

```suggestion
								"to be distributed on (%s) where this Kubernetes API is no longer supported. Please, "+
```